core/templates/dev/head/components/answer_group_editor_directive.html

```diff
@@ -97,7 +97,7 @@
       + Add Another Possible Answer
     </button>
 
-    <div ng-if="isEditable && isCurrentInteractionTrainable()">
+    <div ng-if="isEditable && isCurrentInteractionTrainable() && isMLEnabled()">
```

Comment:
Might be better to have the new check first, as typically && stop checking conditions if any one fails. Like if I have A & B, and A results in false, B isn't checked at all. 
It will work fine as such too!

Reply:
Well, if its fine either way then let's go ahead and merge this. Short-circuiting is not critical, right?

Reply:
Merging (might as well :) )